### PR TITLE
Type Definitions for package project-name

### PR DESCRIPTION
**Pull request by ajilo297**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/project-name/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+// Type definitions for project-name
+// Project: https://github.com/jonschlinkert/project-name
+// Definitions by: Ajil Oommen <https://github.com/ajilo297/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/**
+ * Type Definition for project-name
+ * @param cwd The Base Current Working Directory of project 
+ */
+export default function (cwd: string): string;
```
Comment (DanielRosenwasser) [inline types/project-name/index.d.ts]: This package was declared using `module.exports = ...` or some variant. It should be declared with an `export = ...` like one of the following:

### Variables

```ts
declare var xyz: SomeType;

export = xyz;
```

### Classes

```ts
class Foo {
  // ...
}

export = Foo
```

### Functions

```ts
function foo(): any;

export = foo;
```

To import the entity using a default import, you will need to enable `esModuleInterop`. Alternatively, you can just import using a more appropriate syntax like

```ts
import foo = require("foo");
```

[See more here in the FAQ](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/9f9f6430117d025aef62ed56128deb65508edc86/README.md#a-package-uses-export--but-i-prefer-to-use-default-imports-can-i-change-export--to-export-default).

**Review by DanielRosenwasser — CHANGES_REQUESTED** [review]

File: types/project-name/index.d.ts (hunk 1 above)
Comment (ajilo297) [reply types/project-name/index.d.ts]: I have made the changes

File: types/project-name/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+// Project: https://github.com/jonschlinkert/project-name
+// Definitions by: Ajil Oommen <https://github.com/ajilo297/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export = index;
+/**
+ * Type Definition for project-name
+ * @param cwd The Base Current Working Directory of project
+ */
+declare function index(cwd: string): string;
```
Comment (DanielRosenwasser) [inline types/project-name/index.d.ts]: `cwd` should be optional as per the docs.

File: types/project-name/tslint.json [hunk 3]
```diff
@@ -0,0 +1,80 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "adjacent-overload-signatures": false,
```
Comment (DanielRosenwasser) [inline types/project-name/tslint.json]: Please remove all these toggled lint rules.

**Review by DanielRosenwasser — CHANGES_REQUESTED** [review]

File: types/project-name/tslint.json (hunk 3 above)
Comment (ajilo297) [reply types/project-name/tslint.json]: Fixed.

**Comment by DanielRosenwasser** [comment]
Thank you!

**Pull request merged**